xclim/indices/_agro.py

```diff
@@ -113,3 +101,132 @@ def corn_heat_units(
 
     chu.attrs["units"] = ""
     return chu
+
+
+@declare_units(
+    tasmin="[temperature]",
+    tasmax="[temperature]",
+    thresh_tasmin="[temperature]",
+)
+def biologically_effective_degree_days(
+    tasmin: xarray.DataArray,
+    tasmax: xarray.DataArray,
+    lat: Optional[Union[xarray.DataArray, np.array]] = None,
```

Comment:
```suggestion
    lat: Optional[xarray.DataArray] = None,
```
Would there be a reason to accept a ndarray? Could the user not always wrap it in xarray themself? The Indicator parser wouldn't know what to do with this annotatio, whereas with `Optional[DataArray]` it detects an "OPTIONAL_VARIABLE".

Reply:
I think I figured that supporting any array-type object would be fine. In the test for it, passing a `numpy.array([45])` was fine. I suppose expecting a DataArray makes the most sense though.